src/diffusers/pipelines/audioldm2/pipeline_audioldm2.py

```diff
@@ -407,6 +454,18 @@ def encode_prompt(
                     prompt_embeds = prompt_embeds[:, None, :]
                     # make sure that we attend to this single hidden-state
                     attention_mask = attention_mask.new_ones((batch_size, 1))
+                elif text_encoder.config.model_type == "vits":
```

Comment:
[Use a single variable to check if it's a ViTS model](https://github.com/huggingface/diffusers/pull/5381/files#r1542416708)